veggieseasons/lib/widgets/fade_transition_page.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+
+class FadeTransitionPage<T> extends Page<T> {
```

Comment:
From https://github.com/flutter/samples/blob/3a0a652984e9b974342d172b9f0ffa161d0dcb2f/navigation_and_routing/lib/src/widgets/fade_transition_page.dart with the added restorationId

Reply:
Hmm, maybe [CustomTransitionPage](https://github.com/flutter/packages/blob/main/packages/go_router/example/lib/transition_animations.dart) needs add support for the restorationId, but this LGTM for now.